helper/cluster_test.go

```diff
@@ -0,0 +1,29 @@
+package helper
+
+import (
+	"testing"
+	"time"
+
+	"github.com/shoenig/test/must"
+)
+
+func TestCluster_RandomStagger(t *testing.T) {
+	cases := []struct {
+		name  string
+		input time.Duration
+	}{
+		{name: "positive", input: 1 * time.Second},
+		{name: "negative", input: -1 * time.Second},
+		{name: "zero", input: 0},
+	}
+
+	abs := func(d time.Duration) time.Duration {
+		return Max(d, -d)
+	}
+
+	for _, tc := range cases {
+		result := RandomStagger(tc.input)
+		must.GreaterEq(t, result, 0)
+		must.LessEq(t, result, abs(tc.input))
```

Comment:
Just now realizing I got these parameters backwards, https://github.com/shoenig/test/issues/33

Reply:
fwiw, having the parameters in the same "direction" as the operator feels more intuitive to me. ex. `must.GreaterEq(t, a, b)` reads like it should be checking `a > b`. Whereas for something like `must.Eq` there isn't an obvious expectation vs result in the way the parameters are named anyways.